### PR TITLE
Cleaning up and updating the roadmap.md

### PR DESCRIPTION
**Pull request by Petermarcu**

### CODE REVIEW

**Comment by Petermarcu** [comment]
@richlander , Can you review?

File: roadmap.md [hunk 1]
```diff
@@ -18,27 +20,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
 
 |Milestone|Release Date|
 |---------|------------|
-|1.0      |   June 2016|
+|1.0      |   Spring 2016|
 |1.1      |   Fall 2016|
+|1.2      |   Spring 2016|
```
Comment (svick) [inline roadmap.md]: Should this be 2017?

File: roadmap.md [hunk 2]
```diff
@@ -97,6 +95,10 @@ Microsoft Distro
 
 Microsoft ships multiple .NET Core distros. It is important that Microsoft can successfully ship .NET Core at quality and meet its desired dates.
 
+Other Distros
+================
+.NET Core will ship as part of many Linux distros and we are actively working with key partners in the Linux community to make it natural for .NET Core to get everyone people need it. We are contantly looking to expand our distro support and welcome contributions and collaborations in this direction.
```
Comment (svick) [inline roadmap.md]: > to make it natural for .NET Core to get everyone people need it

I don't understand this part. Was it supposed to be "everywhere people need it"?

Comment (Petermarcu) [inline roadmap.md]: Good catches! Fixed.

File: roadmap.md [hunk 3]
```diff
@@ -18,27 +20,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
...
 |Milestone|Release Date|
 |---------|------------|
-|1.0      |   June 2016|
+|1.0      |   Spring 2016|
 |1.1      |   Fall 2016|
+|1.2      |   Spring 2017|
 
 Planned 1.1 features
 --------------------
+- Lot's of fixes based on feedback.
```
Comment (mairaw) [inline roadmap.md]: Lot's -> Lots

File: roadmap.md [hunk 4]
```diff
@@ -18,27 +20,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
...
-TBD
-===
+Notes:
 
-There are several areas of the product that are TBD. This designation doesn't mean that they are not important, just that there is no plan in place yet. Contributions are welcome.
+- The 1.0 release is accompanied with a preview version of the Visual Studio and command-line tooling. The Visual Studio tooling for 1.0 through 1.2 based on MSBuild and csproj should be publicly available in Fall 2016 and reach RTM quality in Spring 2017. 
 
-- OS package manager (e.g. apt-get, brew), and broader Linux distro support.
-- Create low-level FX APIs for Linux (e.g. daemon support) and macOS, much like exists for Windows (e.g. Windows registry).
+- The [ASP.NET Core roadmap](https://github.com/aspnet/Home/wiki/Roadmap) articulates the ASP.NET projects's roadmap and dates.
```
Comment (mairaw) [inline roadmap.md]: projects's -> projects'

File: roadmap.md [hunk 5]
```diff
@@ -97,6 +95,10 @@ Microsoft Distro
 
 Microsoft ships multiple .NET Core distros. It is important that Microsoft can successfully ship .NET Core at quality and meet its desired dates.
 
+Other Distros
+================
+.NET Core will ship as part of many Linux distros and we are actively working with key partners in the Linux community to make it natural for .NET Core to go everywhere people need it. We are contantly looking to expand our distro support and welcome contributions and collaborations in this direction.
```
Comment (mairaw) [inline roadmap.md]: typo: contantly

**Comment by Petermarcu** [comment]
Thanks @mairaw . @richlander look good?

File: roadmap.md [hunk 6]
```diff
@@ -18,27 +20,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
 
 |Milestone|Release Date|
 |---------|------------|
-|1.0      |   June 2016|
+|1.0      |   Spring 2016|
```
Comment (richlander) [inline roadmap.md]: I think we can just delete 1.0 at this point. The roadmap isn't intended to be historical in nature.

**Comment by richlander** [comment]
LGTM modulo one piece of feedback.

Thanks!

**Comment by Petermarcu** [comment]
Thanks all!

**Pull request merged**